src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+
+import java.util.Scanner;
+
+import duke.TaskManager;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printInstructions() {
+        System.out.println("LIST OF ALL COMMANDS:");
+        System.out.println("todo + \"task name\" to add a task");
+        System.out.println("deadline + \"task name\" + /by \"task deadline\" to add a task with a deadline");
+        System.out.println(
+                "event + \"event name\" + /from \"event start time\" + /to \"event finish time\" to add an event with a stant and finish time");
+        System.out.println("list to list all events and tasks available");
+        System.out.println("mark + \"task number\" to mark a task");
+        System.out.println("mark + \"task number\" to unmark a task");
+    }
+
+    public static void printHorizontalLine() {
+        for (int i = 0; i <= 30; i++) {
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
+    public static void falseInput() {
+        System.out.println("Sorry Duke could not understand your input :> please follow the instructions");
+        printInstructions();
+    }
```

Comment:
Method's name should start with a verb for example announceFalseInput. Same with firstWord method below